act/io/arm.py

```diff
@@ -841,12 +841,12 @@ def read_arm_mmcr(filenames):
     # read it in with xarray
     multi_ds = []
     for f in filenames:
-        nc = Dataset(f, 'a')
-        # Change heights name to range to read appropriately to xarray
-        if 'heights' in nc.dimensions:
-            nc.renameDimension('heights', 'range')
+        nc = Dataset(f, 'r')
         if nc is not None:
             ds = xr.open_dataset(xr.backends.NetCDF4DataStore(nc))
+            # Change heights name to range to read appropriately to xarray
+            if 'heights' in ds.dims:
+                ds = ds.rename_dims({"heights": "range"})
```

Comment:
We are using heights later on for some calculations, this might break

Reply:
@zssherman Well, it doesn't, and I've tested it. 
As it stands, ARM currently holds only the b1 MMCR files, which have that dimension. The issue was with read-only files. Other than that, everything remained the same.
You can try the few lines of code in the issue I opened and see that it works.

Reply:
Yeah i was curious on why we are checking for nans later on etc for height and if that would break if we had range, but I see we check for range as well. Thanks for the clarification.